docs/protocol-changes.rst

```diff
@@ -132,6 +132,11 @@ Changes
   * Optional *cp_height* argument added to
     :func:`blockchain.block.header` and :func:`blockchain.block.headers`
 
+New methods
+-----------
+
+  * :func:`blockchain.transaction.get_id_from_pos`
```

Comment:
Can you rename it to id_from_pos?  The initial "get" has been dropped from recent method names.